common/util/updateNumSegments.py

```diff
@@ -6,7 +6,7 @@
 args = parser.parse_args()
 
 sys.path.append(args.pyqlabpath)
-from Libraries import instrumentLib
+from QGL.ChannelLibrary import channelLib
```

Comment:
actually, I don't know if this is used anywhere. This was a copy and paste error